gulpfile.js

```diff
@@ -0,0 +1,143 @@
+var autoprefixer = require("gulp-autoprefixer");
+var changed = require("gulp-changed");
+var connect = require("gulp-connect");
+var eslint = require("gulp-eslint");
+var gulp = require("gulp");
+var gutil = require("gulp-util");
+var less = require("gulp-less");
+var minifyCSS = require("gulp-minify-css");
+var path = require("path");
+var uglify = require("gulp-uglify");
+var webpack = require("webpack");
+
+var dirs = {
+  src: "./src",
+  js: "./src/js",
+  dist: "dist",
+  styles: "./src/css",
+  img: "./src/img",
+  imgDist: "img"
+};
+
+var files = {
+  mainJs: "main",
+  mainJsDist: "main",
+  mainLess: "main",
+  mainCssDist: "main",
+  index: "index.html"
+};
+
+var webpackConfig = {
+  entry: dirs.js + "/" + files.mainJs + ".jsx",
+  output: {
+    path: path.resolve(dirs.dist),
+    filename: files.mainJsDist + ".js"
+  },
+  module: {
+    loaders: [
+      {
+        test: /\.jsx$/,
+        loader: "babel-loader",
+        exclude: /node_modules/
```

Comment:
Yeah – as I said I wanted to keep this PR as close as possible to what we have in Broccoli for the sake of comparison; let's gather all our wishes for post-merge phase, OK? I will create an issue now so we can use to keep track